docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -121,4 +121,37 @@ ansible-doc
 .. code-block:: bash
 
   ansible-doc -t module -l
-  #...
\ No newline at end of file
+  #...
+
+ansible-config
+==============
+The ansible-config command in Ansible is used to view or modify Ansible configuration settings. 
+
```

Comment:
Sorry I didn't think of this sooner. So Don asked for examples and what we have below is mostly just a repeat of what is already on the cli command docs.

So instead, I did a little searching around for what might be good actual examples and I found two:

* `ansible-config init --disabled > ansible.cfg`  - Creates a sample ``ansible.cfg`` file with all default values. The ``--disabled`` option preceeds each setting with a comment.
* `ansible-config dump -v --only-changed` - Displays only config settings that are not the default values.   The `-v` option displays the locatoin of the ``ansible.cfg`` file used.

So I recommend removing from this line to the end of the file and replace it with the above two examples. Thanks